solarforecastarbiter/reports/template.py

```diff
@@ -32,14 +51,35 @@ def _get_render_kwargs(report, dash_url, with_timeseries):
             div = """<div class="alert alert-warning">
   <strong>Warning</strong> Failed to make timeseries and scatter plots
   from stored data. Try generating report again.
-</div>
-            """
+</div>"""
         kwargs['timeseries_script'] = script
         kwargs['timeseries_div'] = div
     return kwargs
 
 
 def get_template_and_kwargs(report, dash_url, with_timeseries, body_only):
+    """Returns the jinja2 Template object and a dict of template variables for
+    the report.
+
+    Parameters
+    ----------
+    report: :py:class:`solarforecastarbiter.datamodel.Report`
+    dash_url: str
+        URL of the Solar Forecast arbiter dashboard to use when building links.
+    with_timeseries: bool
+        Whether or not to include timeseries plots.
+    body_only: bool
+        When True, returns a standalone report template with the required
```

Comment:
Opposite. When true, return the div, otherwise return standalone with ``<html>`` and ``<head>``